src/header/Header.js

```diff
@@ -126,27 +91,23 @@ Header.defaultProps = {
 };
 
 const styles = StyleSheet.create({
-  innerContainer: {
-    flex: 1,
+  container: {
```

Comment:
Also maybe if we wanna be fancy we could add platform styling to the container. So on android it could have the raised look like material design toolbar.